active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,336 @@
+
+XAML
+```XAML
+<Grid>
+        <!-- DataGrid Layout -->
+        <controls:PagerControl x:Name="MainPagerControl"
+                DisplayMode="Auto"
+                NumberOfPages="10"
+                FirstButtonVisibility="None"
+                PreviousButtonVisibility="AlwaysVisible"
+                NextButtonVisibility="AlwaysVisible"
+                LastButtonVisibility="None"
+                Prefix="Page"
+                Suffix="of 10"/>
+        </controls:PagerControl>
+</Grid>
+```
+C#
+(insert code sample)
+```C#
+```
+
+## API Notes
+| Name | Description| Default | 
+|:---:|:---|:---|
+| PagerDisplayMode | Enum that contains 4 values (Auto, ComboBox, NumberBox, ButtonPanel) to represent the look of the pager control. When Auto is selected, the display mode will be ComboBox if the NumberOfPages property is less than 11 otherwise it will be NumberBox. | Auto |
+| PagerButtonVisibilityBehavior | Enum that contains 4 values (Auto, AlwaysVisible, HiddenOnEdge, None) that allows the app developer to hide or show the four edge buttons. HiddenOnEdge will remove the appropriate buttons if the selected page is the last or first page. When the last page is selected, the next and last buttons will be disabled and same when the first page is selected for the first and previous page. When Auto is selected, the visibility mode will be AlwaysVisible. | Auto |
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1 |
+| First, Previous, Next, and Last ButtonCommand | Specially handle the button pressed event for when the end user selects the buttons. | N/A |
+| First, Previous, Next, and Last Style | Give the developer the option to customize the style by changing the text or glyph for the edge buttons.| N/A |
+| ButtonPanelShowFirstAndLast | Boolean to display the ellipses and the first and last index of the numerical button panel display mode. | True |
+| SelectedIndex | The index that is currently selected. It will default to the first index. | 0 |
+| PrefixText | Developer can change the prefix text for the combobox and number box display mode options. | "Page" |
+| SuffixText | Developer can change the suffix text for the combobox and number box display mode options. | "of (NumberofPages)". If NumberOfPages is infinite, display nothing |
+| PagerControlPageChangedEvent | Event that is fired when the end user selects a button from the number panel, any of the 4 directional buttons, or selects an option from the combobox or enters a number in the numberbox. This event will return the index number that the end user selected. If the end user enters a number greter than the number of pages this event will return the last page. If the end user enters a number less than 1 the event will return 1. | N/A |
+
+### Design Components and Functionality
```

Comment:
This isn't something that should go into the API pages, so for this spec template should go in the appendix.